library_simulation/simulation_runs/20220801_00/README.md

```diff
@@ -0,0 +1,35 @@
+# Library simulation run 2022-08-01
+
+Parameters for Jupyter NB:
+
+```python
+AUTHOR_DIST_SHAPE = 3
+MAX_NUM_AUTHORS = 300
+
+PUBLISHER_DIST_SHAPE = 1
+MAX_NUM_PUBLISHERS = 15
+
+TITLES_DIST_SHAPE = 1
+NUM_TITLES = 1000
+
+NUM_PATRONS = 1000
+```
+
+## Raw Output
+
+You can load the raw output (i.e., the results of loading the TSV files without changing column or table names) by using `books_n_patrons.sql`. **This file is schema agnostic**. This means that by default, it just loads the required tables and sequences into the `public` schema of your `mathesar` database. Further, it drops any table named `books_sim` or `patrons_sim` in that schema, as well as any dependencies, then recreates them. Use with caution.
```

Comment:
It would be good to point to the other README file for instructions on how to load a file, it's not immediately obvious how to load the file if you're only looking at this file.

Reply:
Good idea. Added.